lang/en/aspect_ratio.php

```diff
@@ -6,8 +6,8 @@
 	| Aspect Ratio lines.
 	|--------------------------------------------------------------------------
 	*/
-	'5by4' => '5/4 (instagram portrait)',
-	'4by5' => '4/5 (instagram landscape)',
+	'5by4' => '5/4 (instagram landscape)',
+	'4by5' => '5/4 (instagram portrait)',
```

Comment:
I have to admit that I have no idea why they would both be called 5/4???

Reply:
oups.